views/install.sh.erb

```diff
@@ -171,6 +171,13 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
+  platform=$ID
+  if test "x$platform" = "x"; then
+    platform="linux"
+  fi
```

Comment:
Fair enough. I'll look into this and see what degree of translation is necessary/appropriate.